Explain different attestations and add missing info for kwctl (#462)
kubewarden/docs: docs/tutorials/verifying-kubewarden.md

    audit-scanner-attestation-amd64-checksum.txt
Verified OK

$ sha256sum -c audit-scanner-attestation-amd64-checksum.txt
audit-scanner-attestation-amd64-provenance.json: OK
audit-scanner-attestation-amd64-sbom-451fac2e52226302ff449bfe053b3831fd93409b4dad24581b6121cc24daa2c2.json: OK
```

Now that the files integrity is verified, you can inspect the SBOM and Provenance files.

## Helm charts

You can find our Helm charts in our `https://` traditional Helm repository under
https://charts.kubewarden.io.

The same Helm charts are signed via Sigstore's keyless signing, and pushed to an

> OCI repository that can hold both the Helm chart, its signature, and its

Reviewer: ```suggestion
OCI repository that holds both the Helm chart, its signature, and its
```

You can then verify that the cert in the returned json contains the correct
issuer, subject, and `github_workflow_repository` extensions.

The SBOMs are signed and published in the [GitHub
Release](https://github.com/kubewarden/kwctl/releases) of the project.

> The provenance attestation for `kwctl` can be verified by using [`gh

Reviewer: ```suggestion
The provenance attestation for `kwctl` is verified by using [`gh
```
